### PR TITLE
feat(local-video): convert to react

### PR DESCRIPTION
**Pull request by virtuacoplenny**
torture test updates: https://github.com/jitsi/jitsi-meet-torture/pull/92. The updates cover both the local and remote video element conversion.

### CODE REVIEW

**Comment by virtuacoplenny** [comment]
Force pushed a couple changes to fix some temasys issues. They won't necessarily affect local video thumbnail display but they would be needed for remote video thumbnails.

**Comment by yanas** [comment]
Local video looks broken for me in Chrome:
screenshot 2017-07-11 18 04 15

**Comment by virtuacoplenny** [comment]
CSS should be fixed now. Looks like I forgot to bring over some style changes while building on top of the branch and porting over the temasys fixes.

File: react/features/base/media/components/web/VideoTrack.js [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+ */
+class VideoTrack extends AbstractVideoTrack {
+    static defaultProps = {
+        /**
+         * Dispatch an action when the video starts playing.
+         */
+        triggerOnPlayingUpdate: true
+    };
+
+    static propTypes = {
```
Comment (virtuacoplenny) [inline react/features/base/media/components/web/VideoTrack.js]: Sure, I moved over triggerOnPlayingUpdate with a default value of true. I don't think id would be of any interest... The only interest web has is for torture tests.

**Comment by virtuacoplenny** [comment]
Looks like the torture tests are going to fail for the same reason so I'll address those soon.

**Comment by virtuacoplenny** [comment]
Jenkins test this please.

File: react/features/base/media/components/web/VideoTrack.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+ *
+ * @extends AbstractVideoTrack
+ */
+class VideoTrack extends AbstractVideoTrack {
+    /**
+     * Default values for {@code VideoTrack} component's properties.
+     *
+     * @static
+     */
+    static defaultProps = {
```
Comment (yanas) [inline react/features/base/media/components/web/VideoTrack.js]: It doesn't look like you're using the defaultProps or am I missing something?

Comment (virtuacoplenny) [inline react/features/base/media/components/web/VideoTrack.js]: They are used but probably aren't needed anymore. They just make sure className and id aren't undefined but I'm sure react knows how to handle that.

**Pull request merged**